### PR TITLE
Fixes #11527 - Error when loading org file when cfengine and org layers are enabled

### LINKED ISSUE #11527: Error when loading org file when cfengine and org layers are enabled
#### Description :octocat:

When loading org files with the cfengine layer and org layer enabled there is an
error in the /Messages/ buffer.

```
File mode specification error: (file-error Cannot open load file No such file or directory ob-cfengine) [3 times]
```

#### Reproduction guide :beetle:
- Enable cfengine and org layer
- Start Emacs
- Open an org file

*Observed behaviour:* :eyes: :broken_heart:

The /Messages/ buffer contains an error:

```
File mode specification error: (file-error Cannot open load file No such file or directory ob-cfengine) [3 times]
```

*Expected behaviour:* :heart: :smile:

There should not be an error.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 25.2.2
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 901f35849)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm emacs-lisp neotree cfengine org org-config git github)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: layers/+tools/cfengine/packages.el [hunk 1]
```diff
@@ -40,6 +40,6 @@
     :defer t))
 
 (defun cfengine/pre-init-org ()
-  (when (configuration-layer/package-used-p 'org)
+  (when (configuration-layer/layer-usedp 'org)
```
Comment (d12frosted) [inline layers/+tools/cfengine/packages.el]: `configuration-layer/package-usedp` is alias for `configuration-layer/package-used-p`. So I think that we should stick to the original function (left side).

File: layers/+tools/cfengine/packages.el [hunk 2]
```diff
@@ -40,6 +40,6 @@
...
     (spacemacs|use-package-add-hook org
-      :post-config (add-to-list 'org-babel-load-languages '(cfengine . t)))))
+      :post-config (add-to-list 'org-babel-load-languages '(cfengine3 . t)))))
```
Comment (d12frosted) [inline layers/+tools/cfengine/packages.el]: I am not using `cfengine` but quick glancing shows that this is a right change.

File: layers/+tools/cfengine/packages.el (hunk 1 above)
Comment (d12frosted) [reply layers/+tools/cfengine/packages.el]: My bad. `layer` vs `package`

**Comment by d12frosted** [comment]
Thank you :+1: Cherry-picked into `develop`, you can safely delete your branch.

Resulting commits are e09414c2b and bd4e0815c. Note that I've changed `configuration-layer/layer-usedp` to `configuration-layer/layer-used-p`.

**Comment by nickanderson** [comment]
Thank you!

**Pull request closed without merging**